### DISCUSSION TITLE
Confused about how to go back from token selection mode to line selection mode

### DISCUSSION THREAD

**Discussion by ilyagr** [ki-editor/ki-editor#312] [Q&A] [answered]
I pressed `t` and I'm now in the token selection mode. The docs say that the binding for the line selection mode is `l`, but `l` is also bound to "next token" now. It's unclear what I can do. `L` or `u` also do not work.

I'm sure there is a solution; I think it should be documented better. I'd also consider allowing switching selection modes at any time, and not having selection mode keys overlapping with navigation keys (though I don't understand the philosophy enough to judge how reasonable that request is).

Thanks for experimenting with a new and interesting binding system!

**Comment by wongjiahau** [comment, marked as answer]
Sorry the line navigation is bound to `e`, the docs are outdated for this part, I’ll update it.

**Comment by wongjiahau** [comment]
And yes, no selection mode keybindings should overlap with movement keybindings.

**Comment by wongjiahau** [comment]
I’ve updated the docs, thanks for trying it out!

Feel free to let me know if you encounter any issues

**Reply by ilyagr** [reply]
Thank you!
